install

```diff
@@ -21,5 +21,5 @@ is_ok
 
 # Then install with pip will be inside that virtual environment
 note "Installing requirements.."
-pip install -qIr requirements.txt
+pip install -qIr "${app}/requirements.txt"
```

Comment:
Install won't work correctly via a relative URL without this bad boy.

Reply:
yeah didn't test it :(.. cool merging.